writeKML.js

```diff
@@ -54,7 +54,20 @@ module.exports = {
         const kmlpaths = [];
         const links = {};
 
+        // Loop through nodes list, generate KML Placemark for each
         update.nodes.forEach(node => {
+
+            // Get band string from first digit of frequency field
+            var freq1 = node.meshrf && String(node.meshrf.freq)[0] || 'None';
+            const BAND = {
+                "None": "None",
+                "5": "5ghz",
+                "2": "2ghz",
+                "3": "3ghz"
+            };
```

Comment:
Move BAND outside the loop.

Reply:
Yep... of course, not sure why I didn't do that.